src/fixtures/emergencyContacts.fixture.ts

```diff
@@ -0,0 +1,16 @@
+import { IEmergencyContact } from "../entities/IEmergencyContact";
+
+export const emergencyContactsFixture: IEmergencyContact[] = [
+  {
+    id: "5ffd1b86-d347-49e2-b821-4550c72666c1",
+    fullName: "Lady Hamilton",
+    telephoneNumber: "02392 856621",
+    alternativeTelephoneNumber: "02392 856622",
+  },
+  {
+    id: "3851e8c7-6e4e-4827-ab8f-b904f845582f",
+    fullName: "Neil Hamilton",
+    telephoneNumber: "04392 856626",
+    alternativeTelephoneNumber: "04392 856625",
+  },
+];
```

Comment:
These `.fixture.ts`s are not new, just moved from other areas of the application where test data existed in to a single `fixtures/` folder.

Reply:
We discussed keeping these with the files that actually use them, just like the tests are. But that may not scale well, possibly why this move was made. Happy either way.